src/resources/postcss/views/skeleton/_single-event.pcss

```diff
@@ -357,19 +390,8 @@
 	display: flex;
 	justify-content: space-between;
 	
-	a {
-		&:before,
-		&:after {
-			content: '';
-			display: inline-block;
-			height: 20px;
-			vertical-align: middle;
-			width: 11px;
-		}
-		
-		span {
-			display: none;
-		}
+	a span {
```

Comment:
What is this hiding? Generic tags with `display: none` make me uneasy...

Reply:
@Camwyn that's hiding the right angle quote, unfortunately we can't touch the templates to remove it in this project 
![Screen Shot 2021-02-12 at 4 59 22 PM](https://user-images.githubusercontent.com/5049893/107791172-e664a200-6d53-11eb-975f-577c5461ae45.png)